diff --git a/governance/GovernorUpgradeable.sol b/governance/GovernorUpgradeable.sol
index 052dd7444672323f13f876e7bbeb68086613654d..c674b7558605966b413e75da76810a0ace9e46b2 100644
--- a/governance/GovernorUpgradeable.sol
+++ b/governance/GovernorUpgradeable.sol
@@ -64,7 +64,7 @@ abstract contract GovernorUpgradeable is Initializable, ContextUpgradeable, ERC1
     // keccak256(abi.encode(uint256(keccak256("openzeppelin.storage.Governor")) - 1)) & ~bytes32(uint256(0xff))
     bytes32 private constant GovernorStorageLocation = 0x7c712897014dbe49c045ef1299aa2d5f9e67e48eea4403efa21f1e0f3ac0cb00;
 
-    function _getGovernorStorage() private pure returns (GovernorStorage storage $) {
+    function _getGovernorStorage() internal pure returns (GovernorStorage storage $) {
         assembly {
             $.slot := GovernorStorageLocation
         }
diff --git a/governance/TimelockControllerUpgradeable.sol b/governance/TimelockControllerUpgradeable.sol
index 5860963e4486e3c45fd906a563a678701d421a4b..5878029ba4b798e635a52608a65dc8ccc0fe497e 100644
--- a/governance/TimelockControllerUpgradeable.sol
+++ b/governance/TimelockControllerUpgradeable.sol
@@ -38,7 +38,7 @@ contract TimelockControllerUpgradeable is Initializable, AccessControlUpgradeabl
     // keccak256(abi.encode(uint256(keccak256("openzeppelin.storage.TimelockController")) - 1)) & ~bytes32(uint256(0xff))
     bytes32 private constant TimelockControllerStorageLocation = 0x9a37c2aa9d186a0969ff8a8267bf4e07e864c2f2768f5040949e28a624fb3600;
 
-    function _getTimelockControllerStorage() private pure returns (TimelockControllerStorage storage $) {
+    function _getTimelockControllerStorage() internal pure returns (TimelockControllerStorage storage $) {
         assembly {
             $.slot := TimelockControllerStorageLocation
         }
diff --git a/governance/extensions/GovernorCountingSimpleUpgradeable.sol b/governance/extensions/GovernorCountingSimpleUpgradeable.sol
index 755ef0e0503f1653135cb3cb3d310c6b3e4d28d4..91f33bf7380b4accbfd655cd8f846d408a4e1311 100644
--- a/governance/extensions/GovernorCountingSimpleUpgradeable.sol
+++ b/governance/extensions/GovernorCountingSimpleUpgradeable.sol
@@ -70,6 +70,11 @@ abstract contract GovernorCountingSimpleUpgradeable is Initializable, GovernorUp
         return (proposalVote.againstVotes, proposalVote.forVotes, proposalVote.abstainVotes);
     }

+    function againstVotesCount(uint256 proposalId) public view virtual returns (uint256) {
+        GovernorCountingSimpleStorage storage $ = _getGovernorCountingSimpleStorage();
+        return $._proposalVotes[proposalId].againstVotes;
+    }
+
     /// @inheritdoc GovernorUpgradeable
     function _quorumReached(uint256 proposalId) internal view virtual override returns (bool) {
         GovernorCountingSimpleStorage storage $ = _getGovernorCountingSimpleStorage();
